liblmdb-sys/src/lib.rs

```diff
@@ -8,23 +8,22 @@ use libc::{c_int, c_uint, c_void, c_char, size_t};
 #[cfg(any(target_os = "macos", target_os = "ios", target_os = "linux",
           target_os = "freebsd", target_os = "dragonfly",
           target_os = "android"))]
-mod os {
+pub mod os {
```

Comment:
I don't think you need to make this `mod` public - re-export of `mdb_mode_t` actually happens a bit higher in this file https://github.com/vhbit/lmdb-rs/blob/master/liblmdb-sys/src/lib.rs#L5